runtime/tests/error_backtrace1.c

```diff
@@ -25,7 +25,7 @@ void func2() {
 void func1() {
     TaggedPtr taggedErr = _bal_error_construct(makeString("Func1 error"), 26);
     ErrorPtr ep = (ErrorPtr)taggedToPtr(taggedErr);
-    _bal_error_backtrace_print(ep);
+    _bal_error_backtrace_print(ep, 1);
```

Comment:
I don't see how these tests work. How are you checking that the output is correct?

Reply:
Fixed